### PR TITLE
[Do not MERGE yet] RHDM-251 - wildfly 11 upgrade

### PR DESCRIPTION
**Pull request by mareknovotny**

### CODE REVIEW

File: optaplanner-webexamples/pom.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
   <url>https://www.optaplanner.org</url>
 
   <properties>
-    <version.org.wildfly>10.1.0.Final</version.org.wildfly>
+    <version.org.wildfly>11.0.0.Final</version.org.wildfly>
```
Comment (ge0ffrey) [inline optaplanner-webexamples/pom.xml]: Has it been tested that the optaplanner-webexamples still work? Especially their arquillain test?

We need to move this version.org.wildfly to kie-parent or ip-bom. But I am even more worried about the relationship with the arquillian versions - that stuff is really brittle.

**Comment by mareknovotny** [comment]
hey @ge0ffrey, I am not aware of optaplanner-webexamples repository, please point me where it is to try. Btw i am on pto, so I would check that later next week that ;)

**Comment by ge0ffrey** [comment]
@mareknovotny optaplanner-webexamples is a module in this repository. It builds a simple war file to deploy on WildFly and has an arquillian test.

**Comment by mareknovotny** [comment]
ok, I ran full downstream build on CI. If the examples does not run in that I will look at it manual verification on Monday when I came back.

**Pull request merged**